src/js/elm.js

```diff
@@ -6,10 +6,10 @@ const Elm = require('../elm/Main.elm');
 const app = Elm.Main.fullscreen();
 const db = new Dexie('stimmy_things');
 
-app.ports.initCarousel.subscribe(function() {
-  window.requestAnimationFrame(function() {
-    var carouselElement = document.getElementsByClassName('makecarousel')[0];
-    var flkty = new Flickity(carouselElement, {
+app.ports.initCarousel.subscribe(() => {
+  window.requestAnimationFrame(() => {
```

Comment:
I think we should abstract the functions that the different `subscribe` take into their own modules. For example having a `flickity.js` and a `hotspots.js`. Otherwise this file is going to become unwieldy.